static-website/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/azure-cli:2.0.47
+
+LABEL version="1.0.0"
+LABEL maintainer="Microsoft Corporation"
+LABEL com.github.actions.name="Deploy to Azure Storage for Static Website Hosting"
+LABEL com.github.actions.description="GitHub Action for deploying a static website on Windows / Linux"
+LABEL com.github.actions.icon="cloud"
+LABEL com.github.actions.color="blue"
+
+ENV GITHUB_ACTION_NAME="Deploy to Azure Storage for Static Website Hosting"
+
```

Comment:
Would you not be needing this step `az extension add --name storage-preview` ?

Reply:
Worked without it. But didn't notice this feature; Now that I look at it, might be better to use `az storage blob service-properties update --static-website ...`, no?

Reply:
Didn't need it, relied on the the container being named `$web`. But I didn't notice this feature, so now that I look at it, will definitely be better to use the storage preview extension.
Thanks!